### PR TITLE
[FLINK-16297] Remove unnecessary indents and blank lines in code blocks

### PR DESCRIPTION
**Pull request by KarmaGYZ**
## What is the purpose of the change

Remove unnecessary indents and blank lines in code blocks.

## Brief change log

Remove unnecessary indents and blank lines in code blocks

## Verifying this change

Build the doc and check the relevant code blocks.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? no

cc @zentol

### CODE REVIEW

**Comment by zentol** [comment]
That's a lot of whitespace...

File: docs/dev/batch/index.md [hunk 1]
```diff
@@ -588,9 +588,9 @@ val output: DataSet[(Int, String, Double)] = input.sum(0).min(2)
       <td>
         <p>Returns the distinct elements of a data set. It removes the duplicate entries
         from the input DataSet, with respect to all fields of the elements, or a subset of fields.</p>
-      {% highlight scala %}
-         data.distinct()
-      {% endhighlight %}
+{% highlight scala %}
```
Comment (zentol) [inline docs/dev/batch/index.md]: In this file there's also some unnecessary whitespace in the Java Dataset Aggregate transformation.

File: docs/dev/batch/index.zh.md [hunk 2]
```diff
@@ -588,9 +588,9 @@ val output: DataSet[(Int, String, Double)] = input.sum(0).min(2)
       <td>
         <p>Returns the distinct elements of a data set. It removes the duplicate entries
         from the input DataSet, with respect to all fields of the elements, or a subset of fields.</p>
-      {% highlight scala %}
-         data.distinct()
-      {% endhighlight %}
+{% highlight scala %}
```
Comment (zentol) [inline docs/dev/batch/index.zh.md]: same as above likely

**Comment by KarmaGYZ** [comment]
> That's a lot of whitespace...

Yes. That's why I open a new ticket instead of fixing it in FLINK-15562.

Thanks for the review. I've updated the PR. @zentol

**Comment by zentol** [comment]
You missed the chinese document

**Comment by KarmaGYZ** [comment]
> You missed the chinese document (docs/dev/batch/index.zh.md)

Oops. Fixed.
